components/casemap/src/casemapper.rs

```diff
@@ -146,16 +147,18 @@ impl CaseMapper {
     /// as a `LanguageIdentifier` (usually the `id` field of the `Locale`) if available, or
     /// `Default::default()` for the root locale.
     ///
-    /// This function performs legacy head adjustment behavior when [`HeadAdjustment::Adjust`] is set. See
+    /// This function performs "adjust to cased" leading adjustment behavior when [`LeadingAdjustment::Auto`] or [`LeadingAdjustment::ToCased`]
+    /// is set. Auto mode is not able to pick the "adjust to letter/number/symbol" behavior as this type does not load
+    /// the data to do so, use [`TitlecaseMapper`] if such behavior is desired. See
```

Comment:
Suggestion: Make the reference to [`TitlecaseMapper`] more prominent. Most people who want titlecasing should use that API.